awesome.c

```diff
@@ -559,13 +559,14 @@ exit_help(int exit_code)
     FILE *outfile = (exit_code == EXIT_SUCCESS) ? stdout : stderr;
     fprintf(outfile,
 "Usage: awesome [OPTION]\n\
-  -h, --help             show help\n\
-  -v, --version          show version\n\
-  -c, --config FILE      configuration file to use\n\
-      --search DIR       add a directory to the library search path\n\
-  -k, --check            check configuration file syntax\n\
-  -a, --no-argb          disable client transparency support\n\
-  -r, --replace          replace an existing window manager\n");
+  -h, --help                show help\n\
+  -v, --version             show version\n\
+  -c, --config FILE         configuration file to use\n\
+      --search DIR          add a directory to the library search path\n\
+  -k, --check               check configuration file syntax\n\
+  -a, --no-argb             disable client transparency support\n\
+  -m, --screen auto|manual  enable or disable automatic screen detection\n\
```

Comment:
Not clear what that means.. and what the default is.

Reply:
What do you propose?

Reply:
No idea now, but let's just keep it open / in mind.

Reply:
I changed it to `on` and `off`